### ISSUE TITLE
spec 'erase' param on /account/deactivate

### ISSUE THREAD

**Issue by richvdh** [matrix-org/matrix-spec#297]
https://github.com/matrix-org/matrix-js-sdk/pull/649 added an optional 'erase' boolean to the `/account/deactivate` API

**Comment by Johennes** [comment]
It looks like v1.13 has this parameter in the spec now: https://spec.matrix.org/v1.13/client-server-api/#post_matrixclientv3accountdeactivate

**Comment by richvdh** [comment]
Indeed, fixed by https://github.com/matrix-org/matrix-spec/pull/1730
